group_project/assignment_2.py

```diff
@@ -1,6 +1,15 @@
 def all_in_string(a_string, s1, s2, s3):
-    pass
-
+    count = 0    
+    if s1 in a_string:
+        count += 1
+    
+    if s2 in a_string:
+            count += 1
+    
+    if s3 in a_string:
+                count += 1
```

Comment:
Indentation seems to be way off here. Do you know if you switched to tabs?

Reply:
I like that you used the += operator, because you simplified it, by not repeating the count variable.